README.md

```diff
@@ -57,7 +57,7 @@ You can check whether the command exist by execution `which ssh` in the terminal
 
 ## Customizations
 
-The extensions support two customizations formats:
+The extensions support two customizations formats, that you can add to `.devcontainers/devcontainer.json`:
```

Comment:
Devcontainers allow one of the following three paths:
- devcontainer.json
- .devcontainer/devcontainer.json
- .devcontainer/*/devcontainer.json

To be correct and not confuse anyone, I'd rewrite it as `..., that you can add to your devcontainer.json`